examples/full-example/.mendelrc

```diff
@@ -17,8 +17,8 @@ variation-config:
       - partner_C
 
 route-config: # Mendel-middleware related
-  variation: /mendel/:variations/:bundle # Route pattern for variational assets during development
-  hash: /mendel/:hash/:bundle # Route pattern for variational assets during production
+  variation: /mendel/:variations/:bundle.js # Route pattern for variational assets during development
```

Comment:
I think adding extension should be a job for the outlet. Agreed it should not be part of configuration. pathToRegexp sometimes breaks with .js, since . means any character. Express works around it to make config easier, but we don't use express, we use pathToRegex directly, right? Lets avoid all this complexity.